docker-testing/docker-compose-postgres.yml

```diff
@@ -0,0 +1,16 @@
+version: '2.4'
+services:
+  postgres:
+    extends:
+        file: docker-compose.common.yml
```

Comment:
Is there any reason to separate this in the `common.yml` if we are already separating the files, and in the common the services are separated too? What is the added value of the common here?

Reply:
None really.  Just ended up that way because that's how it is in channels.  I'll reorganize into one file each.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/2000/commits/b5026d8631b50055cc0c6fe01b63534d817e70f2